src/main/java/uk/nhs/hee/trainee/details/service/ProgrammeMembershipService.java

```diff
@@ -310,37 +299,15 @@ public boolean isPilot2024(String traineeTisId, String programmeMembershipId) {
     String managingDeanery = programmeMembership.getManagingDeanery();
     LocalDate startDate = programmeMembership.getStartDate();
     LocalDate dayBefore01082024 = LocalDate.of(2024, 7, 31);
-    LocalDate dayAfter31102024 = LocalDate.of(2024, 11, 1);
     if ((PILOT_2024_LOCAL_OFFICES_ALL_PROGRAMMES.stream()
         .anyMatch(lo -> lo.equalsIgnoreCase(managingDeanery)))
-        && (startDate.isAfter(dayBefore01082024) && startDate.isBefore(dayAfter31102024))) {
+        && (startDate.isAfter(dayBefore01082024))) {
       return true;
     }
 
-    if (managingDeanery.equalsIgnoreCase("Yorkshire and the Humber")
-        && (startDate.isAfter(dayBefore01082024) && startDate.isBefore(dayAfter31102024))
-        && programmeMembership.getCurricula().stream().noneMatch(
-        c -> c.getCurriculumSpecialty().equalsIgnoreCase("General Practice"))) {
-      return true;
-    }
-
-    if (managingDeanery.equalsIgnoreCase("South West")
-        && (startDate.isAfter(dayBefore01082024) && startDate.isBefore(dayAfter31102024))
-        && programmeMembership.getCurricula().stream().noneMatch(
-        c -> c.getCurriculumSpecialty().equalsIgnoreCase("General Practice"))) {
-      return true;
-    }
-
-    LocalDate dayAfter31082024 = LocalDate.of(2024, 9, 1);
-    return managingDeanery.equalsIgnoreCase("North West")
-        && (startDate.isAfter(dayBefore01082024) && startDate.isBefore(dayAfter31082024))
-        && (programmeMembership.getCurricula().stream().anyMatch(c ->
-        PILOT_2024_NW_SPECIALTIES.stream().anyMatch(
-            s -> s.equalsIgnoreCase(c.getCurriculumSpecialty())))
-        || programmeMembership.getProgrammeName()
-        .equalsIgnoreCase("Cardio-thoracic surgery (run through)")
-        || programmeMembership.getProgrammeName()
-        .equalsIgnoreCase("Oral and maxillo-facial surgery (run through)"));
+    LocalDate dayBefore01022025 = LocalDate.of(2025, 1, 31);
+    return (managingDeanery.equalsIgnoreCase("Thames Valley")
```

Comment:
We could simplify and merge with the above condition to eliminiate the `PILOT_2024_LOCAL_OFFICES_ALL_PROGRAMMES` list.

```java
LocalDate notificationsEpoch = managingDeanery.equalsIgnoreCase("Thames Valley") ? LocalDate.of(2025, 1, 31) : LocalDate.of(2024, 10, 31)

return startDate.isAfter(notificationsEpoch);
```

Reply:
We still need to exclude North East though, which is why I kept the list